cmd/proxy/sctp_proxy.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"io"
+	"log"
+	"net"
+	"sync"
+
+	"github.com/ishidawataru/sctp"
+)
+
+// SCTPProxy is a proxy for SCTP connections. It implements the Proxy interface to
+// handle SCTP traffic forwarding between the frontend and backend addresses.
+type SCTPProxy struct {
+	listener     *sctp.SCTPListener
+	frontendAddr *sctp.SCTPAddr
+	backendAddr  *sctp.SCTPAddr
+}
+
+// NewSCTPProxy creates a new SCTPProxy.
+func NewSCTPProxy(frontendAddr, backendAddr *sctp.SCTPAddr) (*SCTPProxy, error) {
+	listener, err := sctp.ListenSCTP("sctp", frontendAddr)
+	if err != nil {
+		return nil, err
+	}
+	// If the port in frontendAddr was 0 then ListenSCTP will have a picked
+	// a port to listen on, hence the call to Addr to get that actual port:
+	return &SCTPProxy{
+		listener:     listener,
+		frontendAddr: listener.Addr().(*sctp.SCTPAddr),
+		backendAddr:  backendAddr,
+	}, nil
+}
+
+func (proxy *SCTPProxy) clientLoop(client *sctp.SCTPConn, quit chan bool) {
+	backend, err := sctp.DialSCTP("sctp", nil, proxy.backendAddr)
+	if err != nil {
+		log.Printf("Can't forward traffic to backend sctp/%v: %s\n", proxy.backendAddr, err)
```

Comment:
please change this to use logrus package.
es: `logrus.WithError(err).Errorf("[...]")`

Reply:
Currently, `cmd/proxy` package does not use logrus. 
Not sure why, probably for reducing dependency.